src/components/ServerTechnologiesFeatures/JavascriptFeatures/index.tsx

```diff
@@ -72,6 +73,12 @@ export class JavascriptFeatures extends React.Component {
             href: 'https://www.npmjs.com/package/@openfeature/config-cat-provider',
             svg: ConfigCat,
           },
+          {
+            title: 'DevCycle Provider',
+            description: 'The official DevCycle OpenFeature Provider',
+            href: 'https://www.npmjs.com/package/@devcycle/openfeature-nodejs-provider',
+            svg: DevCycleNoFillSvg,
```

Comment:
```suggestion
            svg: DevCycleNoFillSvg,
            vendorOfficial: true,
```

This adds a little checkbox indicating this is maintained by the vendor, not a community contribution.